image_layers/map-image-layer-sublayer-visibility/src/main/java/com/esri/samples/map_image_layer_sublayer_visibility/MapImageLayerSublayerVisibilitySample.java

```diff
@@ -75,27 +74,38 @@ public void start(Stage stage) {
       controlsVBox.getChildren().addAll(citiesBox, continentsBox, worldBox);
       controlsVBox.getChildren().forEach(c -> ((CheckBox) c).setSelected(true));
 
-      // create a ArcGISMap with the a BasemapType Topographic
+      // create a map view
+      mapView = new MapView();
+
+      // create a ArcGISMap with the a BasemapType Topographic and set it to the map view
```

Comment:
we can "plain english" this comment a little more
```suggestion
      // create an ArcGISMap with the topographic basemap and set it to the map view
```